include/eve/module/math/constant/invcbrt_pi.hpp

```diff
@@ -7,10 +7,38 @@
 //==================================================================================================
 #pragma once
 
-#include <eve/module/core.hpp>
+#include <eve/arch.hpp>
+#include <eve/traits/overload.hpp>
+#include <eve/module/core/decorator/core.hpp>
 
 namespace eve
 {
+template<typename Options>
+struct invcbrt_pi_t : constant_callable<invcbrt_pi_t, Options, downward_option, upward_option>
+{
+  template<typename T, typename Opts>
+  static EVE_FORCEINLINE constexpr T value(eve::as<T> const&, Opts const&)
+  {
+    if constexpr(std::same_as<element_type_t<T>, float>)
+    {
+      if constexpr(Opts::contains(upward2))        return T(0x1.5d95ep-1);
+      else if constexpr(Opts::contains(downward2)) return T(0x1.5d95dep-1);
+      else                                         return T(0x1.5d95ep-1);
+    }
+    else
+    {
+      if constexpr(Opts::contains(upward2))        return T(0x1.5d95df6bd2aeep-1);
+      else if constexpr(Opts::contains(downward2)) return T(0x1.5d95df6bd2aedp-1);
```

Comment:
It seems to me like `upward` and `downward` should be one flag `up, down, default` or just `up, down` since they are mutually exclusive.

The interface would still be `[eve::upward]` but then the second one would override the first one.

Reply:
This is true and should be done this way. I will modify so it works this way.
In fact, they are part of the `upward`, `downward` , `to_nearest` and `toward_zero` which are all mutually exclusive.
We will change this